ddtrace/contrib/internal/ray/patch.py

```diff
@@ -287,6 +287,28 @@ def traced_actor_method_call(wrapped, instance, args, kwargs):
         return wrapped(*args, **kwargs)
 
 
+def traced_get(wrapped, instance, args, kwargs):
+    """
+    Trace the calls of ray.get
+    """
+    if tracer.current_span() is None:
+        tracer.context_provider.activate(_extract_tracing_context_from_env())
+
+    with long_running_ray_span(
+        "ray.get",
+        service=os.environ.get(RAY_SUBMISSION_ID),
+        span_type=SpanTypes.RAY,
+        child_of=tracer.context_provider.active(),
+        activate=True,
+    ) as span:
+        span.set_tag_str(SPAN_KIND, SpanKind.PRODUCER)
+        timeout = kwargs.get("timeout")
+        if timeout is not None:
+            span.set_tag_str("ray.get.timeout_s", str(timeout))
+        _inject_ray_span_tags_and_metrics(span)
```

Comment:
We might be interested in having info about size at least as ray.put is doing.

Additionally, to get args it is better to use` get_argument_value` (see ray.put)